### ISSUE TITLE
Failed resolving constructor for class 'com.nativescript.image.DraweeView'

### ISSUE THREAD

**Issue by keerl** [nativescript-community/ui-image#33]
Updated to the current version and now I am getting the following error on Android:

```
JS: ERROR Error: java.lang.Exception: Failed resolving constructor for class 'com.nativescript.image.DraweeView' with 1 parameters. Check the number and type of arguments.
JS: Primitive types need to be manually wrapped in their respective Object wrappers.
JS: If you are creating an instance of an inner class, make sure to always provide reference to the outer `this` as the first argument.
```

Using it in Angular, but I don't know if that is related to this issue.

**Comment by farfromrefug** [comment]
@keerl thanks fixed in 4.0.13

**Comment by keerl** [comment]
@farfromrefug nice, that fixed it. Thanks for the quick response!
